src/libsystemd/sd-network/network-util.c

```diff
@@ -24,6 +24,7 @@ static const char* const link_operstate_table[_LINK_OPERSTATE_MAX] = {
         [LINK_OPERSTATE_OFF] = "off",
         [LINK_OPERSTATE_NO_CARRIER] = "no-carrier",
         [LINK_OPERSTATE_DORMANT] = "dormant",
+        [LINK_OPERSTATE_DEGRADED_CARRIER] = "degraded-carrier",
```

Comment:
hmm, why call this "carrier" if it typically means there's a carrier missing?

maybe a different name would be better, for example "incomplete"? (because not all slaves are ready yet) or "partial" or "deficient"? @keszybz any other suggestions?